### PR TITLE
Update layouting.jl

### PR DESCRIPTION
**Pull request by logankilpatrick**
Fixed the docstrings for `vbox` and `hbox`.

### CODE REVIEW

**Comment by logankilpatrick** [comment]
@asinghvi17 does this look good?

File: src/layouting/layouting.jl [hunk 1]
```diff
@@ -176,15 +176,15 @@ ispixelcam(x) = false
 """
     vbox(scenes...; parent = Scene(clear = false), kwargs...)
 
-Lay the given Scenes out on the vertical axis.  For example, two Scenes `vbox`ed
+Attach the given Scenes together on the vertical axis.  For example, two Scenes `vbox`ed
```
Comment (asinghvi17) [inline src/layouting/layouting.jl]:
```suggestion
Box the scenes together on the vertical axis.  For example, two Scenes `vbox`ed
```

The reason I say this is that I don't want to give the wrong impression - there's no attachment per se, at least not yet.

File: src/layouting/layouting.jl [hunk 2]
```diff
@@ -176,15 +176,15 @@ ispixelcam(x) = false
...
 will be placed side-by-side.
 """
 vbox(plots::Transformable...; kw_args...) = vbox([plots...]; kw_args...)
 """
     hbox(scenes...; parent = Scene(clear = false), kwargs...)
 
-Lay the given Scenes out on the horizonral axis.  For example, two Scenes `vbox`ed
-will be placed side-by-side.
+Attach the given Scenes together on the horizontal axis.  For example, two Scenes `hbox`ed
```
Comment (asinghvi17) [inline src/layouting/layouting.jl]:
```suggestion
Box the scenes together on the horizontal axis.  For example, two Scenes `hbox`ed
```

File: src/layouting/layouting.jl [hunk 3]
```diff
@@ -176,15 +176,15 @@ ispixelcam(x) = false
...
+will be placed one on top of the other.
```
Comment (asinghvi17) [inline src/layouting/layouting.jl]:
```suggestion
will be placed one on top of the other.

--------------------
--                --
--    Scene 1     --
--                --
--------------------
--------------------
--                --
--    Scene 2     --
--                --
--------------------
```

File: src/layouting/layouting.jl [hunk 4]
```diff
@@ -176,15 +176,15 @@ ispixelcam(x) = false
 """
     vbox(scenes...; parent = Scene(clear = false), kwargs...)
 
-Lay the given Scenes out on the vertical axis.  For example, two Scenes `vbox`ed
+Attach the given Scenes together on the vertical axis.  For example, two Scenes `vbox`ed
 will be placed side-by-side.
```
Comment (asinghvi17) [inline src/layouting/layouting.jl]:
```suggestion
will be placed side-by-side.
--------------------  --------------------
--                --  --                --
--    Scene 1     --  --    Scene 2     --
--                --  --                --
--------------------  --------------------

**Comment by logankilpatrick** [comment]
Should be done!

**Comment by SimonDanisch** [comment]
Thank you :)

**Pull request merged**